eurovision/notebooks/story.ipynb

```diff
@@ -142,7 +142,7 @@
    "outputs": [],
    "source": [
     "## Read in data from the Kaggle dataset\n",
-    "votes_1975_2019 = pd.read_excel('https://github.com/KatrionaGoldmann/Eurovision_TDS/raw/story_notebook/data/eurovision_song_contest_1975_2019.xlsx', engine='openpyxl')\n",
+    "votes_1975_2019 = pd.read_excel('https://github.com/KatrionaGoldmann/Eurovision_TDS/raw/main/data/eurovision_song_contest_1975_2019.xlsx', engine='openpyxl')\n",
```

Comment:
Is there a reason not to use a relative URL here (e.g. `../../data/eurovision_song_contest_1975_2019.xlsx`)?

Probably the whole structure will be revised before it gets published, so it's a academic at this stage, but there are quite a few other places that use relative URLs and so I'm now wondering whether they should be more like this.